assets/content/documentation/H2d/7. Layers.md

```diff
@@ -0,0 +1,57 @@
+# Layers
+
+The Layers module provides additional functionality to the Sprite class, allowing for the ordering of Sprites. If you've ever used Flash, OpenFL or Starling, then you can envisage Layers to be a little like the MovieClip object from those libraries.
+
```

Comment:
Might be slightly confusing : in Heaps every Sprite is a container with a children list (like in Flash).

Layers are a special class in Heaps which treats its children as grouped by "layer" id , which guarantees that every children added to a given layer N is over all children in layer N-1 and under children in layer N+1